exporters/auto/README.md

```diff
@@ -6,16 +6,31 @@
 
 OpenTelemetry Java SDK provides an `autoconfigure` module where all configuration can be done entirely through Java system properties or environment variables.
 
-To leverage the GCP extensions to this, simply add the following dependency:
+To leverage the GCP extensions to this, simply add the exporter-auto dependency to your project. 
 
-```
+*We also provide a `shaded` variant of the exporter-auto which is the recommended way for autoconfiguration if you are using the auto-exporter as a java agent extension.*
+
+#### Maven
+```xml
 <dependency>
   <groupId>com.google.cloud.opentelemetry</groupId>
   <artifactId>exporter-auto</artifactId>
-  <version>0.20.0-alpha</version>
+  <version>0.25.1-alpha</version>
+  <!-- Add the classifier if you explicitly wish to use the shaded variant -->
+  <classifier>shaded</classifier>
 </dependency>
 ```
 
+#### Gradle
+```groovy
+implementation "com.google.cloud.opentelemetry:exporter-auto:0.25.1-alpha"
+```
+
+To use the shaded variant with Gradle,
```

Comment:
Consider using https://github.com/ryandens/javaagent-gradle-plugin#application-plugin-integration to show how to use the shaded variant as an agent (i.e. what we recommend) rather than adding it to the application classpath.

You can write up similar instructions for Maven by adapting https://stackoverflow.com/questions/14777909/specify-javaagent-argument-with-maven-exec-plugin, but it's a lot more verbose. It's up to you whether you want to do that.

Reply:
I will add those instructions.

Reply:
Created a separate issue to track this - https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/issues/246